Orange/tests/test_classification.py

```diff
@@ -77,10 +77,10 @@ def test_value_from_probs(self):
         clf = learn(t)
         clf.ret = Model.Probs
         y2 = clf(x, ret=Model.Value)
-        self.assertTrue(y2.shape == (nrows,))
+        self.assertEqual(y2.shape, (nrows,))
         y2, probs = clf(x, ret=Model.ValueProbs)
-        self.assertTrue(y2.shape == (nrows, ))
-        self.assertTrue(probs.shape == (nrows, 2))
+        self.assertEqual(y2.shape, (nrows, ))
```

Comment:
Well, it's still [WIP]. :) I didn't realize that we had so much to clean up in unit tests. It was about time that somebody starts cleaning this mess.